src/embind/embind.js

```diff
@@ -1913,11 +1916,15 @@ var LibraryEmbind = {
   ) {
     var rawArgTypes = heap32VectorToArray(argCount, rawArgTypesAddr);
     methodName = readLatin1String(methodName);
+    var humanName = methodName;
+    if(methodName == "@@iterator"){
+        methodName = Symbol.iterator;
```

Comment:
Symbol.iterator is the method name used for iterators in ES6.  The @@iterator is just the string that tells embind to use that Symbol.  @@iterator looked like a good convention for the string, based on the "@@iterator" useage here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols